### PR TITLE
Upgrade kuromojin

### PR DESCRIPTION
**Pull request by hata6502**

### CODE REVIEW

File: src/no-dropping-i.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 // LICENSE : MIT
 "use strict";
-import { RuleHelper } from "textlint-rule-helper";
-import kuromojin from "kuromojin";
+const { RuleHelper } = require("textlint-rule-helper");
+const kuromojin = require("kuromojin");
```
Comment (azu) [inline src/no-dropping-i.js:4]: Why use `require`?

**Review by azu — COMMENTED** [review]
Thanks for PR.

I've commented

File: src/no-dropping-i.js (hunk 1 above)
Comment (hata6502) [reply src/no-dropping-i.js:4]: When I use `import kuromojin from "kuromojin";`, the following error occured in `yarn test`. 

```
  1) no-dropping-i
       見てます。:
     TypeError: Cannot read property 'tokenize' of undefined
      at /home/hata/textlint-rule-no-dropping-i/src/no-dropping-i.js:23:30
      at /home/hata/textlint-rule-no-dropping-i/node_modules/@textlint/kernel/src/task/promise-event-emitter.ts:27:35
      at Array.forEach (<anonymous>)
      at RuleTypeEmitter.PromiseEventEmitter.emit (node_modules/@textlint/kernel/src/task/promise-event-emitter.ts:26:38)
      at Controller.enter (node_modules/@textlint/kernel/src/task/textlint-core-task.ts:157:53)
      at Controller.__execute (node_modules/@textlint/ast-traverse/src/index.ts:51:31)
      at Controller.traverse (node_modules/@textlint/ast-traverse/src/index.ts:132:28)
      at TextLintCoreTask.startTraverser (node_modules/@textlint/kernel/src/task/textlint-core-task.ts:152:28)
      at TextLintCoreTask.start (node_modules/@textlint/kernel/src/task/linter-task.ts:43:14)
      at /home/hata/textlint-rule-no-dropping-i/node_modules/@textlint/kernel/src/task/task-runner.ts:28:18
      at new Promise (<anonymous>)
      at Function.TaskRunner.process (node_modules/@textlint/kernel/src/task/task-runner.ts:16:16)
      at LinterProcessor.process (node_modules/@textlint/kernel/src/linter/linter-processor.ts:55:27)
      at TextlintKernel._parallelProcess (node_modules/@textlint/kernel/src/textlint-kernel.ts:155:14)
      at /home/hata/textlint-rule-no-dropping-i/node_modules/@textlint/kernel/src/textlint-kernel.ts:83:25
```

And, it seems `src/no-dropping-i.js` uses CommonJS `module.exports`. 

```
module.exports = function(context) {
```

So, I rewrote to CommonJS `require()`. 

---

Alternative solution: `import * as`

```
import * as kuromojin from "kuromojin";
```

---

いつも Twitter 経由で JSer.info を見させてもらっています。
本年はありがとうございました。

Comment (azu) [reply src/no-dropping-i.js:4]: `tokenize` is named exports.
We should use named import to import `tokenize`.

`import { tokenize } from "kuromoji"`

Comment (azu) [reply src/no-dropping-i.js:4]: 📝 We will change `exports` to `export default`, but it requires major updates.

Comment (hata6502) [reply src/no-dropping-i.js]: Fixed it.

**Comment by azu** [comment]
Thanks!

**Pull request merged**